tests/component-tests/pager/js/PagedTableTests.js

```diff
@@ -150,60 +159,113 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
                 }
-            });
+            ]
         };
+    };
 
-        var tooltipTest = function (location) {
-            return function (idx, linkEl) {
-                var linkId = linkEl.id;
-                var link = $(linkEl);
-                sequence.push({
-                    element: link,
-                    jQueryTrigger: "focus"
+    fluid.tests.assertVisibleTips = function (pager, sequence, message, targetIds, expectedTooltip) {
+        sequence.push({
+            event: "{trackTooltips}.events.notifyFocusChange",
+            listener: function () {
+                fluid.tests.tooltip.assertVisible(message, pager, targetIds, null, function (tooltip) {
+                    jqUnit.assertNode("When the page size is " + pager.model.pageSize + ", The contents of the tooltip should be set", expectedTooltip,
+                        $("b", tooltip));
                 });
+            }
+        });
+    };
 
-                if (link.hasClass(pager.pagerBar.options.styles.currentPage)) { // assumption that this will not change between now and then!
-                    assertVisibleTips("There shouldn't be any tooltips visible when the currentPage is focused", []);
-                } else {
-                    assertVisibleTips("Only the tooltip for page link " + (idx + 1) + ", in the " + location + " page bar is visible", [linkId]);
-                }
-                sequence.push({
-                    element: link,
-                    jQueryTrigger: "blur"
-                });
-                assertVisibleTips("There shouldn't be any tooltips visible when none of the pageLinks are focused", []);
-            };
-        };
+    /**
+     * Change the page size
+     */
+    fluid.tests.setPageSize = function (pager, sequence, pageSize) {
+        sequence.push({
+            func: function () {
+                pager.locate("pageSize").val(pageSize);
+            }
+        });
+        sequence.push({
+            element: "#rendered-ioc .flc-pager-page-size",
+            jQueryTrigger: "change"
+        });
+    };
 
-        pageLinksTop.each(tooltipTest("top"));
-        pageLinksBottom.each(tooltipTest("bottom"));
-        return {
-            name: "Pager tooltip tests",
-            tests: {
-                name: "Tooltip visibility",
-                sequence: sequence
+    /**
+     * verify whether the displaying links and their tooltips
+     */
+    fluid.tests.verifyToolTips = function (pager, sequence, pageSize, expectedTooltips) {
+        var expectedNumberOfLinks = Math.ceil(pager.dataModel.length / pageSize);
+        sequence.push({
+            func: function () {
+                pager.locate("pageSize").val(pageSize);
+                jqUnit.assertEquals("When the page size is " + pager.model.pageSize + ", Top pageLinks",
+                    expectedNumberOfLinks,
+                    $("#rendered-ioc .flc-pager-top .flc-pager-links .flc-pager-pageLink a").length);
```

Comment:
You should not use jQuery directly like this. This will limit the ability to reuse this function for another test. You should make use of the Dom Binder that the component supplies to look up any elements. In this case you'll probably need to go through the pageBar subcomponents to access the elements. So something like:

```javascript
pager.pagerBar.locate("pageLinks").length;
pager["pagerBar-1"].locate("pageLinks").length;
```